packages/terra-clinical-result/tests/jest/flowsheet-result-cell/__snapshots__/FlowsheetResultCell.test.jsx.snap

```diff
@@ -288,59 +312,134 @@ exports[`FlowsheetResultCell should render an error when given no result data 1`
 `;
 
 exports[`FlowsheetResultCell should render an error with bad data 1`] = `
-<div
-  className="flowsheet-result-cell padding-compact"
->
-  <div
-    className="primary-display error"
-    key="ClinicalResultDisplay-Error"
-  >
-    <InjectIntl(ResultError) />
-  </div>
-</div>
+<FlowsheetResultCell
+  intl={
+    Object {
+      "defaultFormats": Object {},
+      "defaultLocale": "en",
+      "formatDate": [Function],
+      "formatHTMLMessage": [Function],
+      "formatMessage": [Function],
+      "formatNumber": [Function],
+      "formatPlural": [Function],
+      "formatRelative": [Function],
+      "formatTime": [Function],
+      "formats": Object {},
+      "formatters": Object {
+        "getDateTimeFormat": [Function],
+        "getMessageFormat": [Function],
+        "getNumberFormat": [Function],
+        "getPluralFormat": [Function],
+        "getRelativeFormat": [Function],
+      },
+      "locale": "en",
+      "messages": null,
+      "now": [Function],
+      "onError": [Function],
+      "textComponent": "span",
+      "timeZone": null,
+    }
+  }
+  paddingStyle="compact"
+  resultDataSet={Array []}
+/>
```

Comment:
This looks different from the previous snapshot. I think you may need a `.dive()` to get the inner content of the component.

Reply:
Good catch. Fixed: https://github.com/cerner/terra-clinical/pull/863/files#diff-8493253d24c22468a23f3598cb31f39b5c0bcbee2a62810fe9db1e5f3b6e6516R552-R600